### DISCUSSION TITLE
ModuleNotFoundError: No module named 'carla.libcarla'

### DISCUSSION THREAD

**Discussion by vagaiprem** [carla-simulator/carla#6724] [General]
Dear All
              I am getting the following error in conda environment ( Windows) :

PS C:\prem\software\CARLA_0.9.14\WindowsNoEditor\PythonAPI\examples> python .\manual_control.py
Traceback (most recent call last):
  File ".\manual_control.py", line 83, in <module>
    import carla
  File "C:\Users\Admin\anaconda3\envs\Carla_1\lib\site-packages\carla\__init__.py", line 8, in <module>
    from .libcarla import *
ModuleNotFoundError: No module named 'carla.libcarla'

anyone give solution plz help me

**Comment by Youngzipper** [comment]
have you found the solution?

**Comment by ouahid-ex9** [comment]
Hi,
I think you should specify in the command line where the python file dist .egg exists.
In my case, i do export this variable:
export PYTHONPATH=$PYTHONPATH:$CARLA_ROOT/PythonAPI/carla/dist/carla-0.9.14-py3.10-linux-x86_64.egg
Search for the equivalent in windows, and add the full path of the file .egg

**Comment by ioioiogjk** [comment]
hello，have you found the solution?I meet the same problem.I am using the VScode in win10.
  File "d:\BaiduNetdiskDownload\carla_0.9.15\WindowsNoEditor_9.15\PythonAPI\examples\automatic_control.py", line 58, in <module>
    import carla
ModuleNotFoundError: No module named 'carla'
